great-idea-website/css/index.css

```diff
@@ -63,4 +64,137 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Your code starts here! */
\ No newline at end of file
+/* Your code starts here! */
+* {
+	box-sizing: border-box;
+	text-align: center;
+
+	
+}
+*, *:before, *:after{
+	box-sizing: inherit;
+}
+header{
+	text-align: center;
+	width: 100%;
+	
+}
+.container {
+	display: block;
+    /* width: 765px; */
+    height: 19.5px;
+    padding: 2px;    
+    /* vertical-align: bottom; */
+	
+	
+
+}
+nav {
+	display: inline-block;
+    /* display: inline; */
+    width: 450px;
+    height: 19.5px;
+    margin: 1,1,1,1;
+    margin: 2px;
+    vertical-align: bottom;
+    margin-right: 60px;
+	
+	
+	
+}
+a{
+    display: inline;
+    margin-right: 34px;
+    height: 19.5;
+    font-family: inherit;
+    font-size: 7px;
+    /* margin-bottom: 100px; */
+    /* border: 2px solid black; */
+	
+
+}
+.logo {
+	
+    height: inherit;
+    /* border: 2px solid black; */
+    /* margin-top: 5px; */
+    /* display: inline-block; */
+
+}
+.all-cta{
+	text-align: center;
+	display: inline-block;
+	width: 100%;
+	vertical-align: middle;
+
+  
+}
+.ctaContent{
+	display: inline-block;
+	text-align: center;
+	margin-top: 70px;
+	margin-bottom: 60px;
+	margin-left: 35px;
+	height: 136px;
+	width: 172px;
+	vertical-align: middle;
+}
+
+button {
+	margin-top: 12px;	
+	border: 2px solid black;
+}
+
+.cta-img{
+	margin-top: 48px;
+	margin-left: 42px;
+	vertical-align: middle;
+}
+
+.featureAbout{
+	width: 100%;
+    height: 166px;
+    text-align: center;
+    width: 100%;
+}
+.features, .about {
+	margin: 16px;
+	border: auto;
+	display: inline-block;
+    height: 166px;
+	text-align: left;
+    width: 300px;
+}
+
+.SPV{
+	width: 100%;
+    height: 166px;
+    text-align: center;
+    width: 100%;
+}
+.services, .product, .vision{
```

Comment:
great job using the syntax for applying multiple stylings to the same classname.  That being said though, considering that they are all doing the same thing, I would personally just use one generic class like .bottom-text?  It's a totally a personal design choice, but I find that a more simplistic approach is easier to read.